api/resources_portal/migrations/0001_initial.py

```diff
@@ -497,7 +497,7 @@ class Migration(migrations.Migration):
                             ("ORG_REQUEST_REJECTED", "ORG_REQUEST_REJECTED"),
                             ("ORG_INVITE_INVALID", "ORG_INVITE_INVALID"),
                             ("ORG_REQUEST_INVALID", "ORG_REQUEST_INVALID"),
-                            ("MTA_UPLOADED", "MTA_UPLOADED"),
+                            ("SIGNED_MTA_UPLOADED", "SIGNED_MTA_UPLOADED"),
```

Comment:
Why is this migration being updated?

Reply:
Ahhh, my bad. I did a project-wide search and replace and didn't consider that this would be in it. Fixed now.